### PR TITLE
ft: layout implementation

### PR DESCRIPTION
**Pull request by gabrieliyeh**
Initial Tailwind setup and implementation of the new website layout.

### CODE REVIEW

File: .env
Comment (tobi-oye) [inline .env:1]: yikes can we add the .env to .gitignore

**Review by tobi-oye — COMMENTED** [review]
asides the .env lgtm

**Comment by DeolaJ** [comment]
Screenshot 2024-09-22 at 4 04 53 PM

**Comment by DeolaJ** [comment]
Please delete this branch and recreate with a new branch, or squash the commits when you merge?

**Pull request closed without merging**